harris/harris.go

```diff
@@ -0,0 +1,195 @@
+package harris
+
+import (
+	"yi/matrix"
+	"image"
+	"yi/point"
+	"log"
+)
+const k = 0.04
+
+func smoothFilter() matrix.Matrix {
+	result := matrix.Ones(3, 3).Scale(float64(1)/float64(9))
```

Comment:
Ah, thats true. I think when I was looking at this originally I read it as `float64(1), float64(9)`